smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/OpenApiConverterTest.java

```diff
@@ -63,6 +63,7 @@ public void convertsModelsToOpenApi() {
         ObjectNode result = OpenApiConverter.create()
                 .config(config)
                 .convertToNode(testService);
+        String afb = Node.prettyPrintJson(result);
```

Comment:
Looks like this was meant to be taken back out.

Reply:
embarrasing